test/mainnet/invariants.spec.sol

```diff
@@ -218,22 +217,26 @@ contract VaultMainnetInvariantsTest is Test, MainnetActors {
 
         uint256 shares = vault.convertToShares(assets);
         uint256 convertedAssets = vault.convertToAssets(shares);
-        assertApproxEqRel(convertedAssets, assets, 1e14, "Converted assets should equal the original assets");
+        assertApproxEqAbs(convertedAssets, assets, 3, "Converted assets should equal the original assets");
 
         vm.startPrank(alice);
         IERC20(baseAsset).approve(address(vault), assets);
         uint256 depositedShares = vault.depositAsset(baseAsset, assets, alice);
         vm.stopPrank();
 
-        assertApproxEqRel(depositedShares, shares, 1e14, "Deposited shares should equal the converted shares");
+        assertApproxEqAbs(depositedShares, shares, 3, "Deposited shares should equal the converted shares");
 
         // hypothetically allocated 100% to the buffer
         allocateToBuffer(assets);
 
         // Test the previewRedeem function
         uint256 previewedRedeemAssets = vault.previewRedeem(shares);
-        assertApproxEqRel(
-            previewedRedeemAssets, assets, 2e15, "Previewed redeem assets should equal the original assets"
+        // TODO: see why precision is not high here.
+        assertApproxEqAbs(
+            previewedRedeemAssets,
+            assets - FeeMath.feeOnTotal(assets, vault.baseWithdrawalFee()),
+            100000,
```

Comment:
still questionable why this is so high but notably more acceptable than a 0.2% error tolerance which is absurdly high